examples/rt/async/microphone/main.py

```diff
@@ -97,17 +97,21 @@ def handle_partial_transcript(message):
             print("Microphone started - speak now...")
             print("Press Ctrl+C to stop transcription\n")
 
-            await client.transcribe(
-                source=mic,
+            await client.start_session(
```

Comment:
I had a quick check of this in JS and there we just use start, but start_session is good also. I'll leave it to you to decide

Reply:
start is a good name but I think start_session is a bit more descriptive